main.js

```diff
@@ -562,6 +576,11 @@ function generateItemDOMFragment(id) {
     return fragment;
 }
 
+function createLabelHTML(type, itemId, placeholderMessage) {
+    let html = '<p class="cell-' + type + '-label" id="' + type + '-label-' + itemId + '"> ' + placeholderMessage + '</p>'
```

Comment:
Can you use ticks here? (Top left of keyboard). They tend to make stuff like this a ton easier to read. [Check here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for more info.